shotover-proxy/src/message/mod.rs

```diff
@@ -377,26 +402,38 @@ impl Value {
             match spec.col_type.id {
                 ColType::Ascii => Value::Strings(decode_ascii(actual_bytes).unwrap()),
```

Comment:
Shouldn't Ascii have it's own field in Value? Same goes for other Cassandra types that we are currently putting in string. 

@benbromhead

Reply:
yes.
The reason I didnt bring it up in the review was because it was a preexisting issue.
Although thinking about it, it does make sense to resolve it in this PR. (Although still happy for it to go in a follow up)

Reply:
I'll do it in this PR